src/FormsyContext.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { FormsyContextInterface } from './interfaces';
+
+const noFormsyErrorMessage = 'Could not find Formsy Context Provider. Did you use withFormsy outside Formsy ?';
```

Comment:
Unsure if this is clearer, your call:
```suggestion
const noFormsyErrorMessage = 'Could not find Formsy Context Provider. Did you use withFormsy outside <Formsy />?';
```